css/modules/aioseop_module.css

```diff
@@ -1264,6 +1225,48 @@ div.aioseop_notice a.aioseop_dismiss_link {
 	vertical-align: bottom;
 }
 
+.aioseop_tabs.ui-widget-content {
```

Comment:
Do you think this would a good time to create a *-ui.css file for jquery-ui stuff?

Reply:
Maybe if there are more rules. But it would also be a performance hit by loading 2 resources so I don't think it is required at this point in time.